tests/test_models.py

```diff
@@ -1,38 +1,186 @@
-Test cases for YourResourceModel Model
+Test cases for Wishlist Model
 
 """
 import os
 import logging
 import unittest
-from service.models import YourResourceModel, DataValidationError, db
+from service import app
+from service.models import Wishlist, DataValidationError, db
+from tests.factories import WishlistFactory
 
 
+DATABASE_URI = os.getenv(
+    "DATABASE_URI", "postgresql://postgres:postgres@localhost:5432/postgres"
+)
+
 ######################################################################
-#  YourResourceModel   M O D E L   T E S T   C A S E S
+#  Wishlist   M O D E L   T E S T   C A S E S
 ######################################################################
 # pylint: disable=too-many-public-methods
-class TestYourResourceModel(unittest.TestCase):
-    """ Test Cases for YourResourceModel Model """
+class TestWishlist(unittest.TestCase):
+    """ Test Cases for Wishlist Model """
 
     @classmethod
     def setUpClass(cls):
         """ This runs once before the entire test suite """
+        app.config["TESTING"] = True
+        app.config["DEBUG"] = False
+        app.config["SQLALCHEMY_DATABASE_URI"] = DATABASE_URI
+        app.logger.setLevel(logging.CRITICAL)
+        Wishlist.init_db(app)
 
     @classmethod
     def tearDownClass(cls):
         """ This runs once after the entire test suite """
+        db.session.query(Wishlist).delete()  # clean up the last tests
+        db.session.commit()
+        #db.drop_all() #Drops all tables if needed for updating schema
 
     def setUp(self):
         """ This runs before each test """
+        db.session.query(Wishlist).delete()  # clean up the last tests
+        db.session.commit()
 
     def tearDown(self):
         """ This runs after each test """
+        db.session.remove()
 
     ######################################################################
     #  T E S T   C A S E S
     ######################################################################
 
-    def test_example_replace_this(self):
-        """ It should always be true """
-        self.assertTrue(True)
+    def test_create_a_wishlist(self):
+        """It should Create a Wishlist and assert that it exists"""
+        fake_wishlist = WishlistFactory()
+        # pylint: disable=unexpected-keyword-arg
+        wishlist = Wishlist(
+            customer_id=fake_wishlist.customer_id,
+            wishlist_name=fake_wishlist.wishlist_name,
+            created_date=fake_wishlist.created_date,
+        )
+        self.assertIsNotNone(wishlist)
+        self.assertEqual(wishlist.wishlist_id, None)
+        self.assertEqual(wishlist.customer_id, fake_wishlist.customer_id)
+        self.assertEqual(wishlist.wishlist_name, fake_wishlist.wishlist_name)
+        self.assertEqual(wishlist.created_date, fake_wishlist.created_date)
+
+    def test_add_a_wishlist(self):
+        """It should Create a Wishlist and add it to the database"""
+        wishlists = Wishlist.all()
+        self.assertEqual(wishlists, [])
```

Comment:
Technically, we probably don't want to assert that the table is empty as that isn't really an assert on the create() method.  I totally see what you are going for though.  I think there is a principle that you should try to limit the test case to the exact thing you are testing.  I think Prof said what you are doing is okay though, i.e. doing what you need to to ensure the thing functions as designed.  One idea/suggestion:  Instead of asserting == [], maybe take the length of wishlists and then after .create(), assert that the count is one greater than it started.